apidef/oas/middleware.go

```diff
@@ -1492,13 +1544,19 @@ type EndpointPostPlugins []EndpointPostPlugin
 // EndpointPostPlugin contains endpoint level post plugin configuration.
 type EndpointPostPlugin struct {
 	// Enabled activates post plugin.
+	//
+	// Tyk classic API definition: `custom_middleware.go_plugin.disabled` (negated).
```

Comment:
```suggestion
	// Tyk classic API definition: `version_data.versions..extended_paths.go_plugin.disabled`(negated).
```
this is an endpoint level plugin, and not API level.